### PR TITLE
Calendar module excluded events filter

### PR DESCRIPTION
**Pull request by amcolash**
This is a simple filter that will hide certain calendar events from showing up on your mirror. A use case would be like hiding all birthdays from the mirror. It just does a simple `includes` check, so no fancy regex patterns. Should be useful for others!

### CODE REVIEW

**Comment by fewieden** [comment]
depends on which way we want to go #694 includes is an es6 feature

**Comment by amcolash** [comment]
Gotcha, I just did a quick search on W3Schools (I am a Java dev for the most part, where we have `containsIgnoreCase`). A quick search returns a [stackoverflow answer](http://stackoverflow.com/questions/1789945/how-to-check-whether-a-string-contains-a-substring-in-javascript) with a bunch of options we could use. `indexOf` would be outdated, but works on < es6. I have no real preference as I was just interested in simple string matching. I don't think I am the one to make that call.

File: modules/default/calendar/calendar.js [hunk 1]
```diff
@@ -38,7 +38,10 @@ Module.register("calendar", {
 			"De verjaardag van ": "",
 			"'s birthday": ""
 		},
-		broadcastEvents: true
+		broadcastEvents: true,
+		excludedEvents: [
```
Comment (roramirez) [inline modules/default/calendar/calendar.js]: Can be better is use [] instead of [""]

Comment (amcolash) [inline modules/default/calendar/calendar.js]: ah, right. thanks for that! will add that to the pull branch

Comment (amcolash) [inline modules/default/calendar/calendar.js]: fixed with commit 5466e1b

**Comment by MichMich** [comment]
Nice work! Is it possible for you to fix the conflicts?

**Comment by amcolash** [comment]
I probably messed up the CHANGELOG on one of the other pull requests because I merged them before all of the changes were taken upstream.

**Comment by MichMich** [comment]
Thanks for your contributions!

**Pull request merged**